src/functions-and-arrays.js

```diff
@@ -52,14 +151,45 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(arr) {
+
+  let uniqueArr = []
 
+  if (arr.length === 0) {
+    return null
+  }
 
+  for (let i = 0; i < arr.length; i++) {
+    if (i == arr.indexOf(arr[i])) {
+      uniqueArr.push(arr[i])
+    }
+  }
+
+  return uniqueArr
+
+}
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(arr, word) {
+
+  if (arr.length === 0) {
+    return null
+  }
+
+  for (let i = 0; i < arr.length; i++) {
+    
+    if (arr[i] === word) {
+      return true
+    }
+    else {
+      continue
+    }
```

Comment:
aqui se não tem nada abaixo do if, não precisa do else/continue, afinal, o for vai se repetir de qqr forma.
Lembre, break e continue usamos respectivamente quando queremos parar a iteração ou que ele vá para a próxima iteração imediatamente. Mas se não tem mais código ele já ia fazer isso de qqr maneira, certo?